vcsetup.iss

```diff
@@ -3,44 +3,60 @@
 #define public Dependency_Path_NetCoreCheck "InnoDependencyInstaller\dependencies\"
 #include "InnoDependencyInstaller\CodeDependencies.iss"
 
-#define AppID "{33DFCEE8-022C-4C66-A366-79A7415320F2}"
-#define AppName "Volume Control"
-#define AppPublisher "radj307"
-#define AppURL "https://github.com/radj307/volume-control"
-#define AppExeName "VolumeControl.exe"
-#define AppMutex "VolumeControlSingleInstance"
+#define AppID               "{33DFCEE8-022C-4C66-A366-79A7415320F2}"
+#define AppName             "Volume Control"
+#define AppPublisher        "radj307"
+#define CurrentYear         GetDateTimeString('yyyy','','')
+#define StartYearCopyright  "2011"
+#define AppURL              "https://github.com/radj307/volume-control"
+#define AppExeName          "VolumeControl.exe"
+
+#define AppMutex            "VolumeControlSingleInstance"
 ; #define AppVersion "" ;< DEFINE THIS VIA COMMANDLINE (/dAppVersion="VERSION_NUMBER")
 
 [Setup]
-AppId={{#AppID}
+AppId={#AppID}
 AppName={#AppName}
 AppVersion={#AppVersion}
 AppVerName={#AppName} {#AppVersion}
-AppPublisher={#AppPublisher}
+
+AppPublisher=(c){#StartYearCopyright}-{#CurrentYear} {#AppPublisher}
 AppPublisherURL={#AppURL}
 AppSupportURL={#AppURL}
 AppUpdatesURL={#AppURL}
+
+VersionInfoDescription={#AppName} installer
+VersionInfoProductName={#AppName}
+VersionInfoVersion={#AppVersion}
+
+UninstallDisplayIcon={app}\{#AppExeName}
+UninstallDisplayName={#AppName}
+
+ShowLanguageDialog=yes
+UsePreviousLanguage=no
+LanguageDetectionMethod=uilanguage
+
+WizardStyle=modern
+WizardSizePercent=100
+
+;PrivilegesRequiredOverridesAllowed=dialog
```

Comment:
Please uncomment this.  
Personally, I'm on the fence about removing the "Install for all users" prompt. But since I have seen people using it, I'm not going to remove it unless there is a reason to do so.

Reply:
@radj307 
PLease take that for some reason if you use this control when you run installer on top left you didn't have teh info about version of you are  instlling.
If you don't use this control instead you have this info.

Reply:
@radj307 
I forgot to mention that GNU license text is in old 80'chars for line format.
I asked to GNU if there is any problem to use longer lines and separate paragraph with new line.
GNU official answer me that is not important the layout (80 chars for lines or more) but the text  contents.
Of course is not possible to modify the text contents but no problem to change the layout.
Current layout 80chars for lines leave a lot of space empty on the left.

![image](https://github.com/radj307/volume-control/assets/1262554/d177c0dd-6583-4533-b563-977ee13f855d)

Reply:
EWhere can I get
@radj307 
I forgot to mention that GNU license text is in old 80'chars for line format.
I asked to GNU if there is any problem to use longer lines and separate paragraph with new line.
GNU official answer me that is not important the layout (80 chars for lines or more) but the text  contents.
Of course is not possible to modify the text contents but no problem to change the layout.
Current layout 80chars for lines leave a lot of space empty on the left.

![image](https://github.com/radj307/volume-control/assets/1262554/d177c0dd-6583-4533-b563-977ee13f855d)

Reply:
Updaed Inno Setup script.
Just remain to decide about LICENMSE file (no 80 chars limitation for lines).

Reply:
Not sure if it'll be possible to fix this without modifying the LICENSE file (which should be avoided at all costs)